modules/osd-persistent-storage-csi-efs-sts.adoc

```diff
@@ -112,10 +112,11 @@ $ rosa describe cluster \
 +
 [source,terminal]
 ----
-$ aws iam create-role \
+ROLE_ARN=$(aws iam create-role \
   --role-name "<your_cluster_name>-aws-efs-csi-operator" \
   --assume-role-policy-document file://<your_trust_file_name>.json \
-  --query "Role.Arn" --output text
+  --query "Role.Arn" --output text)
+echo $ROLE_ARN
```

Comment:
Per our guidelines [1], we shouldn't have more than one command in a single code block. 

> Do not use more than one command per code block. 

So this would need to be in a separate code block. Or you could consider combining them so that it is still just a single command. I think something like this would work:

```
ROLE_ARN=$(aws iam create-role \
  --role-name "<your_cluster_name>-aws-efs-csi-operator" \
  --assume-role-policy-document file://<your_trust_file_name>.json \
  --query "Role.Arn" --output text
  --query "Role.Arn" --output text); echo $ROLE_ARN
```

Same comment for the below code block

[1] https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc

Reply:
@bergerhoffer Thanks for your input. I agree we could consider combining them so that it is still just a single command. 

Just wondering if we should combine them as you proposed earlier,

ROLE_ARN=$(aws iam create-role \
  --role-name "<your_cluster_name>-aws-efs-csi-operator" \
  --assume-role-policy-document file://<your_trust_file_name>.json \
  --query "Role.Arn" --output text
  --query "Role.Arn" --output text); echo $ROLE_ARN
  
  OR you intended to propose as below
  
  ROLE_ARN=$(aws iam create-role \
  --role-name "<your_cluster_name>-aws-efs-csi-operator" \
  --assume-role-policy-document file://<your_trust_file_name>.json \
  --query "Role.Arn" --output text); echo $ROLE_ARN
  
  Would love to hear your thoughts. Thanks!

Reply:
@bergerhoffer Thanks for your input. I agree we could consider combining them so that it is still just a single command. 

Just wondering if we should combine them as you proposed earlier,

`ROLE_ARN=$(aws iam create-role \
  --role-name "<your_cluster_name>-aws-efs-csi-operator" \
  --assume-role-policy-document file://<your_trust_file_name>.json \
  --query "Role.Arn" --output text
  --query "Role.Arn" --output text); echo $ROLE_ARN`

  
  OR you intended to propose as below
  
  `ROLE_ARN=$(aws iam create-role \
  --role-name "<your_cluster_name>-aws-efs-csi-operator" \
  --assume-role-policy-document file://<your_trust_file_name>.json \
  --query "Role.Arn" --output text); echo $ROLE_ARN`

  
  Would love to hear your thoughts. Thanks!

Reply:
Thanks for your input @bergerhoffer. I agree we could consider combining them so that it is still just a single command.

Just wondering if you are suggesting to make updates as follows (as you proposed earlier),
```
ROLE_ARN=$(aws iam create-role \
  --role-name "<your_cluster_name>-aws-efs-csi-operator" \
  --assume-role-policy-document file://<your_trust_file_name>.json \
  --query "Role.Arn" --output text
  --query "Role.Arn" --output text); echo $ROLE_ARN
``` 

OR if we need to make updates as following,

```
ROLE_ARN=$(aws iam create-role \
  --role-name "<your_cluster_name>-aws-efs-csi-operator" \
  --assume-role-policy-document file://<your_trust_file_name>.json \
  --query "Role.Arn" --output text); echo $ROLE_ARN
``` 

Would like to hear your thoughts. Thank you!

Reply:
I believe the later one seems to be ok. Let me know if you have other thoughts. Thanks!